### PR TITLE
add linea/mantle/bnb/scroll chains to swaap-v2 DEX

### PR DESCRIPTION
**Pull request by borelien**
adding linea/mantle/bnb/scroll chains to swaap-v2 DEX

### CODE REVIEW

**Comment by borelien** [comment]
hey @Hosuke
thanks for taking (again) my PR!
I have some Cannot cast '474440809439857' to INTEGER.
How am I supposed to handle bigint in such cases?

also I just sent the mantle/linea/vault contracts to Dune for approval, so might be not available yet for testing not sure (they were deployed few months ago so Swaap has volume on those chains already)

File: dbt_subprojects/dex/seeds/trades/swaap_bnb_base_trades_seed.csv
Comment (Hosuke) [inline dbt_subprojects/dex/seeds/trades/swaap_bnb_base_trades_seed.csv:1]: We need to add schema for these seed files.

**Comment by borelien** [comment]
We actually submitted the wrong contract few months ago.
I asked a question on discord in #decoding to know if we can replace the old contract with the proper one.
When this is sorted out I will update this PR accordingly, but I think everything is good except this issue (which affect the bnb contract)

**Comment by borelien** [comment]
the contract has been accepted, so I think I'm good on my end.
The CI is not ending though, for some reason

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @borelien

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**